### PR TITLE
Leo's changes to AMD Device Exporter

### PR DESCRIPTION
**Pull request by lpaoletti**
Re-organized main page, edited all topics, and added two new topics.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 project = "AMD Instinct Hub"
 version = "1.0.0"
 release = version
-html_title = f"Device Metrics Exporter"
+html_title = f"AMD Device Metrics Exporter"
```
Comment (randyh62) [inline docs/conf.py:11]: Is this F necessary?

**Review by randyh62 — APPROVED** [review]
Practically explains itself. Thanks for your effort.

File: docs/installation/docker.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
-# Docker
+# Docker installation
 
-## Prerequisites
+This page explains how to install AMD Device Metrics Exporter using Docker. 
+
+## System requirements
 
 - ROCm 6.2.0
```
Comment (farshadghodsian) [inline docs/installation/docker.md:7]: Can we update this to say ROCm 6.2.0 or later?

**Review by farshadghodsian — COMMENTED** [review]
Looks good, just one minor update around adding "ROCm 6.2.0 or later".

**Pull request merged**